src/network/models/network-block.model.ts

```diff
@@ -0,0 +1,19 @@
+/* istanbul ignore file */
+import { ApiProperty } from '@nestjs/swagger';
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+
+import { FromBigNumber } from '~/common/decorators/transformation';
+
+export class NetworkBlockModel {
```

Comment:
IMO we don't require this model. We can directly send in the block number. WDUT @polymath-eric ?

Reply:
I think other things could go into this model though. We could return `23454` and technically its valid JSON, but generally people expect an object to be returned.

My preference would be to leave this. It can get more stuff added like block hash, number of events etc. and will help make us consistent

Reply:
The SDK method should have probably returned the full block for this, not just the number.

We can still do a second query if we want to support a more full model so I don't think its an issue, although not optimal.

Reply:
Object return value is much easier to extend. A raw primitive value is impossible to change without it being a major version change or new endpoint